test/modules/post/test/file.rb

```diff
@@ -185,6 +185,42 @@ def test_binary_files
     end
   end
 
+  def test_path_expansion_nix
+    unless session.platform =~ /win/i
+      it "should expand home" do
+        home1 = expand_path('~')
+        home2 = expand_path('$HOME')
+        home1 == home2 && home1.length > 0
+      end
+
+      it "non-isolated tilde should not expand" do
+        s = '~a'
+        result = expand_path(s)
+        s == result
+      end
+
+      it "mid-string tilde should not expand" do
+        s = '/home/~'
+        result = expand_path(s)
+        s == result
+      end
+
+      it "no env vars should not expand" do
```

Comment:
Naming of this test title could be improved, sounds very double negative here which is confusing.

```suggestion
      it "env vars with invalid naming should not expand" do
```

Reply:
Fixed in ebd927da44f89faf96a4a256f1f9627c91c698e0